listenbrainz/db/model/feedback.py

```diff
@@ -10,13 +10,15 @@ class Feedback(BaseModel):
             user_id: the row id of the user in the DB
             user_name: (Optional) the MusicBrainz ID of the user
             recording_msid: the MessyBrainz ID of the recording
-            score: The score associated with the recording (+1/-1 for love/hate respectively)
+            score: the score associated with the recording (+1/-1 for love/hate respectively)
+            created: (Optional)the timestamp when the feedback record was inserted into DB
     """
 
     user_id: int
     user_name: str = None
     recording_msid: str
     score: int
+    created: datetime = None
```

Comment:
This should be required, no?

Reply:
why is it set to none by default

Reply:
Because we also validate the feedback submitted via API using this model and it doesnt have a created field